src/cli.ts

```diff
@@ -3,34 +3,43 @@
 /* NOTE: calling this file entropy.ts helps commander parse process.argv */
 import { Command, Option } from 'commander'
 import Entropy from '@entropyxyz/sdk'
-import { cliListAccounts } from './flows/manage-accounts/cli'
-import { currentAccountAddressOption, endpointOption, loadEntropy, cliWrite } from './common/utils-cli'
+
+import * as config from './config'
+import { EntropyTuiOptions } from './types'
+import { currentAccountAddressOption, endpointOption, loadEntropy } from './common/utils-cli'
+
+import launchTui from './tui'
+import { entropyAccountCommand } from './account/command'
 import { entropyTransferCommand } from './transfer/command'
 import { entropySignCommand } from './sign/command'
 import { entropyBalanceCommand } from './balance/command'
-import { EntropyTuiOptions } from './types'
-import launchTui from './tui'
-
-const program = new Command()
 
-let entropy: Entropy
+let entropyGlobal: Entropy
 async function setEntropyGlobal (address: string, endpoint: string, password?: string) {
+  console.log('args', address, endpoint, password);
+  
+  let entropy: Entropy = entropyGlobal
```

Comment:
:fire: the function had be modified to `set local entropy` within function (never modify `entropyGlobal`). I think it was a mistake and should be:

```suggestion
let entropy: Entropy
async function setEntropyGlobal (address: string, endpoint: string, password?: string) {
  console.log('args', address, endpoint, password);

```